README.md

```diff
@@ -1,3 +1,278 @@
+<div align="center">
+  <a href="https://github.com/leapfrogtechnology/chill">
+    <img width="128px" src="opensource-logo.png"/> 
+  </a>
+  <br/>
+
 # opensource
 
 Opensource as a culture at Leapfrog Technology.
+
+# Featured List
+List of open source softwares in Leapfrog
+- [sync-db](#sync-db)
+- [sports]()
+- [just-handlebars-helpers]()
+- [async-store]()
+- [chill]()
+- [eslint-config-leapfrog]()
+- [tslint-config-leapfrog]()
+- [terraform-modules]()
+- [Nepali-Date-Picker]()
+- [spanner-orm]()
+- [bulletin-board]()
+- [journal-recommendation]()
+- [image-based-calorie-estimation]()
+- [nepali-image-captioning]()
+- [leapfrogger-react]()
+- [koothrappali]()
+- [Leapicons]()
+- [weather]()
+
+
+## [sync-db](https://github.com/leapfrogtechnology/sync-db)
+Utility to synchronize relational database objects across databases.
+Maintained by Kabir Baidhya, Saugat Acharya
```

Comment:
Don't need anything except for the description. Everything else needs to be updated often. Only include a good description of the project.

Let's make this as minimal as we can.

Reply:
If any project has their separate logo, make sure we link the image too.

Reply:
Understood

Reply:
@mesaugat Also, suggestion on the url placement.

Reply:
Description rakhne ra? how about just linking to the projects github page? and people can read everything from there:
```
## Featured open source projects

- [chill](github.com/leapfrogtechnology/chill)
```

Reply:
Instead we can then create a standard where each project will have the following checklist in their readme
```
Lastest Release Date: Aug 9, 2019
Latest Version: v1.0.0-alpha.2
Community Profile
✔️Description
✔️Readme
❌Code Of Conduct
❌Contribution Guidelines
✔️License
❌Issue Tempalates
❌Pull request template
```

I think this format is too much to maintain.

Reply:
@aviskarkc10 yes it would have been. It is now gone.